app/src/main/java/org/glucosio/android/presenter/AddGlucosePresenter.java

```diff
@@ -119,6 +132,20 @@ public void dialogOnAddButtonPressed(String time, String date, String reading, S
         }
     }
 
+    @Nullable
+    private Number parseReading(@Nullable String reading) {
```

Comment:
it shouldn't be `null` from `dialogOnAddButtonPressed` but i put it there in case it'll be used in other places in the future.